Libraries/LibWeb/DOM/Document.idl

```diff
@@ -144,12 +144,12 @@ interface Document : Node {
 
     // https://w3c.github.io/editing/docs/execCommand/
     // FIXME: [CEReactions] boolean execCommand(DOMString commandId, optional boolean showUI = false, optional (TrustedHTML or DOMString) value = "");
-    [CEReactions] boolean execCommand(DOMString commandId, optional boolean showUI = false, optional DOMString value = "");
-    boolean queryCommandEnabled(DOMString commandId);
-    boolean queryCommandIndeterm(DOMString commandId);
-    boolean queryCommandState(DOMString commandId);
-    boolean queryCommandSupported(DOMString commandId);
-    DOMString queryCommandValue(DOMString commandId);
+    [CEReactions, ImplementedAs=exec_command_binding] boolean execCommand(DOMString commandId, optional boolean showUI = false, optional DOMString value = "");
+    [ImplementedAs=query_command_enabled_binding] boolean queryCommandEnabled(DOMString commandId);
+    [ImplementedAs=query_command_indeterm_binding] boolean queryCommandIndeterm(DOMString commandId);
+    [ImplementedAs=query_command_state_binding] boolean queryCommandState(DOMString commandId);
+    [ImplementedAs=query_command_supported_binding] boolean queryCommandSupported(DOMString commandId);
+    [ImplementedAs=query_command_value_binding] DOMString queryCommandValue(DOMString commandId);
```

Comment:
Instead of all these bindings, can't we just do `is<HTML::HTMLDocument>(*this)` checks in the existing methods? It would keep the IDLs a bit neater.

Reply:
Sure. The downside is that we end up with otherwise unnecessary `MUST` wrapping everywhere, which is why I created these `*_binding` methods.

Reply:
We'll probably run into more exceptions that we need to add in the individual commands once we iron out the differences between browsers, so that's an improvement as far as I'm concerned!